profiles.wordpress.org/public_html/wp-content/plugins/wporg-5ftf-activity-handler/wporg-5ftf-activity-handler.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/*
+Plugin Name: WordPress.org 5ftf Activity Handler
+Plugin URI: http://wordpress.org
+License: GPL2
+Version: 1.1
+Description: Handles saving of last 5ftf contribution.
+*/
+
+defined( 'ABSPATH' ) or die();
+
+if ( ! class_exists( 'WPOrg_5ftf_Activity_Handler' ) ) {
```

Comment:
```suggestion
if ( class_exists( 'WPOrg_5ftf_Activity_Handler' ) ) {
	return;
}
```

This way the rest of the file doesn't have to be indented.

Reply:
Yep.